src/test/java/com/commercetools/project/sync/customer/CustomerSyncerTest.java

```diff
@@ -0,0 +1,51 @@
+package com.commercetools.project.sync.customer;
+
+import static io.sphere.sdk.json.SphereJsonUtils.readObjectFromResource;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.mock;
+
+import com.commercetools.sync.customers.CustomerSync;
+import com.commercetools.sync.customers.utils.CustomerReferenceResolutionUtils;
+import io.sphere.sdk.client.SphereClient;
+import io.sphere.sdk.customers.Customer;
+import io.sphere.sdk.customers.CustomerDraft;
+import io.sphere.sdk.customers.queries.CustomerQuery;
+import java.time.Clock;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.CompletionStage;
+import org.junit.jupiter.api.Test;
+
+class CustomerSyncerTest {
```

Comment:
I would add `// preparation`, `// test` &  `// assertion` sections for each tests to adhere to the convention

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/184/commits/2a922d18bcff9bbfd40f61040edc3cac6fc94010